.gitignore

```diff
@@ -70,9 +70,13 @@ tools/keytools/x64
 tools/keytools/Debug
 tools/keytools/Release
 
-# preseal binary
+# preseal binaries
 tools/preseal/preseal
 tools/preseal/preseal.exe
```

Comment:
Are these *.exe files actually created?  If so, they should be removed during make clean